packages/core/assemblyManager/assembly.ts

```diff
@@ -381,9 +381,9 @@ export default function assemblyFactory(
        */
       async loadPre() {
         const conf = self.configuration
-        const refNameAliasesAdapterConf = conf.refNameAliases?.adapter
-        const cytobandAdapterConf = conf.cytobands?.adapter
-        const sequenceAdapterConf = conf.sequence.adapter
+        const refNameAliasesAdapterConf = conf?.refNameAliases?.adapter
```

Comment:
any reason for additional undefined checks?

Reply:
Yes, TS found that some of those things can be undefined

Reply:
I see that the configuration is a safeReference so that could be indeed possibly undefined, but my editor says self.configuration is type any so wonder if it regressed back to any somehow

Reply:
yeah it did regress back, somehow, let me see if I can improve the types there

Reply:
Actually I am not sure it did regress, I think I may have been using a different version of TS or mst that caught that error

Reply:
can you identify which versions? the repo of course has the lock file, and ts/mst versions haven't changed substantially for awhile